internal/panda_controller/docker_agent.go

```diff
@@ -83,7 +98,7 @@ func (pa *dockerPandaAgent) connectGrpc(ctx context.Context) error {
 }
 
 // See documentation for docker CopyToContainer
```

Comment:
Is this documentation in a file we have or is it from official docker documentation?